src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,39 @@
+package duke;
+
+import duke.command.Command;
+import duke.task.Task;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ParserTest {
+    @Test
+    public void testParserUnknownCommand(){
+        DukeException exception = assertThrows(DukeException.class,
```

Comment:
Cool test case 😄 

Can the name of the test case be clearer though? Something like `parser_unknownCommand`. Can consider this with regards to the rest of the test cases as well.

Reply:
Yes, I have made the changes for the test cases based on the lecture topics.